build-conf/Cobol.properties

```diff
@@ -23,6 +23,12 @@ cobol_objPDS=${hlq}.OBJ
 cobol_dbrmPDS=${hlq}.DBRM
 cobol_BMS_PDS=${team}.BMS.COPY
 
+#
+# data sets used to pass in external dependencies 
+# both for compile and link steps 
+cobol_cpyImpPDS=${hlq}.COPY.IMP
```

Comment:
I think I would name these variables differently, like `cobol_imported_cpyPDS` and `cobol_imported_objPDS`, and the datasets too, like `${hlq}.IMPORT.COPY`.